From 0fdef8547c448ef69d7833b9de89496b68413874 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Wed, 10 Apr 2013 21:13:24 -0700
Subject: [PATCH 1008/1357] HADOOP-9471. hadoop-client wrongfully excludes jetty-util JAR, breaking webhdfs. (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1466763 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 29c964d6bc1b117bbbc78c7278773d6c18bc54a1)

Conflicts:
	hadoop-client/pom.xml
	hadoop-common-project/hadoop-common/CHANGES.txt
---
 hadoop-client/pom.xml |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/hadoop-client/pom.xml b/hadoop-client/pom.xml
index dc0bd0b..aa45396 100644
--- a/hadoop-client/pom.xml
+++ b/hadoop-client/pom.xml
@@ -136,10 +136,6 @@
           <artifactId>jetty</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>org.mortbay.jetty</groupId>
-          <artifactId>jetty-util</artifactId>
-        </exclusion>
-        <exclusion>
           <groupId>tomcat</groupId>
           <artifactId>jasper-runtime</artifactId>
         </exclusion>
-- 
1.7.0.4

